tests/Components/Powertrain/test_SimpleMotor.py

```diff
@@ -0,0 +1,54 @@
+import attr
+from vec import Vector2
+from adr.World import Ambient
+from adr.Components import FreeBody
+from adr.Components.Powertrain import Motor, SimpleMotor
+from adr.Methods.Powertrain.thrust_equations import get_axial_thrust_from_linear_model
+import numpy.testing as npt
+import math
+import pytest
+
+@pytest.fixture
+def motor():
+    motor = SimpleMotor(
+       name='attached_component',
+       mass=1.4,
+       relative_position=Vector2(-0.4, 0.1),
+       relative_angle=math.radians(0),
+       static_thrust = 78,
+       linear_coefficient = 1,
+       distance_origin_to_propeller = 0.3
+    )
+    return motor
+
+@pytest.fixture
+def plane():
+    env = Ambient()
+    plane = FreeBody(
+        name='freebody',
+        type='generic_freebody',
+        mass=23.4,
+        position_cg=Vector2(-0.2, 0.02),
+        pitch_rot_inertia=5.2,
+        ambient=env,
+    )
+    return plane
+
+def test_thrust_center(motor, plane):
+   
+    motor.set_parent(plane)
+    
+    check_thrust_center = Vector2(0.3, 0)
+
+    assert(motor.thrust_center == check_thrust_center)
+
+def test_get_thrust(motor, plane):
+   
+    motor.set_parent(plane)
+    x,y = motor.get_thrust()
+
+    check_thrust_center = Vector2(0.3, 0)
+    check_thrust = Vector2(82.26940810871174, 0)
+
+    npt.assert_array_almost_equal(x, check_thrust)
+    npt.assert_array_almost_equal(y, check_thrust_center)
```

Comment:
Another nice assert would be to incline the plane and assert that the thrust also inclines by the same value (that is, that the implementation assures that the thrust is aligned with the component axis).

Reply:
Sorry, but i'm not quite sure how to implement this.

Reply:
Adding this after your previous code:

```
plane.angle = math.radians(15)

x,y = motor.get_thrust()

check_thrust_center = Vector2(0.3, 0)
check_thrust = Vector2(82.26940810871174, 0)

npt.assert_array_almost_equal(x, check_thrust)
npt.assert_array_almost_equal(y, check_thrust_center)
```

Basically, you are checking if the thrust remains the same if the plane is inclined (15 degrees) or not (0 degrees). It should be, since the get_thrust() method returns the thrust in the motor frame of reference.

Reply:
Perfect. Will upload later a commit with this.